### PR TITLE
Tweaks for using Firebase

### PR DESCRIPTION
**Pull request by liberaliscomputing**
edit index.html and js/game.js to import the Firebase library and to log data in the Firebase database.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
     <link rel="stylesheet" href="node_modules/animate.css/animate.min.css">
     <link href='https://fonts.googleapis.com/css?family=PT+Sans|Fredoka+One|Source+Code+Pro' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/style.css">
+    <script src="https://cdn.firebase.com/js/client/2.4.1/firebase.js"></script>
```
Comment (thomaspark) [inline index.html]: What was the reason for putting this in `head` instead of with the other scripts at line 96?

File: js/game.js [hunk 2]
```diff
@@ -301,6 +305,8 @@ var game = {
       }
     });
 
+    console.log(game.changed);
```
Comment (thomaspark) [inline js/game.js]: You forgot to remove this.

File: js/game.js [hunk 3]
```diff
@@ -389,4 +413,4 @@ var game = {
 
 $(document).ready(function() {
   game.start();
-});
+});
```
Comment (thomaspark) [inline js/game.js]: Can you keep the newline at the end of file?

**Comment by thomaspark** [comment]
Hey @liberaliscomputing, I left a few comments on your PR. Thanks.

**Comment by liberaliscomputing** [comment]
Hey Tom,

I just checked your message. I will re-do PR shortly!

**Pull request merged**